examples/features/authz/README.md

```diff
@@ -38,3 +40,23 @@ Start the client with:
 ```
 go run client/main.go
 ```
+
+### Authorization by watching a policy file
+
+The server can accept an optional `--authz-option filewatcher` flag to set up authorization policy by reading a [policy file](/examples/data/rbac/policy.json), and to look for update on the policy file every 100 millisecond. Having `GRPC_GO_LOG_SEVERITY_LEVEL` environment variable set to `info` will log out the reload activity of the policy every time a file update is detected.
```

Comment:
Nit: s/The server can accept/The server accepts/

Reply:
Updated.